src/plugins/workspace/public/components/delete_workspace_modal/delete_workspace_modal.test.tsx

```diff
@@ -0,0 +1,268 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { DeleteWorkspaceModal, DeleteWorkspaceModalProps } from './delete_workspace_modal';
+import { coreMock } from '../../../../../core/public/mocks';
+import { render, fireEvent, waitFor, cleanup } from '@testing-library/react';
+import { workspaceClientMock } from '../../../public/workspace_client.mock';
+import { OpenSearchDashboardsContextProvider } from '../../../../../plugins/opensearch_dashboards_react/public';
+
+const defaultProps: DeleteWorkspaceModalProps = {
+  onClose: jest.fn(),
+  selectedWorkspace: null,
+  returnToHome: true,
+};
+
+const coreStartMock = coreMock.createStart();
+
+function WrapWorkspaceDeleteModalInContext(
```

Comment:
It seems `WrapWorkspaceDeleteModalInContext` is not a component, can we renaming like `getWrapWorkspaceDeleteModalInContext`?